### PR TITLE
step 3 commit

### PR DESCRIPTION
**Pull request by cjy9306**
please check this out

### CODE REVIEW

**Review by jukqaz — APPROVED** [review]
👍

**Review by korJiusngPark — APPROVED** [review]
good

File: 3/cjy/10_11720.cpp [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	int sum = 0;
+	char arr[101] = { 0 };
+
+	scanf("%d", &count);
+
+	scanf("%s", arr);
+
+	for (int i = 0; i < count; i++)
+	{
+		sum += (arr[i] - 48);
```
Comment (AlvinYou) [inline 3/cjy/10_11720.cpp:15]: 48 equal '0'?

**Review by AlvinYou — APPROVED** [review]
👍

**Review by kor-haru — APPROVED** [review]
👍

**Pull request merged**